src/tinyformat.h

```diff
@@ -1153,6 +1154,7 @@ template<typename... Args>
 std::string format(const std::string &fmt, const Args&... args)
 {
     std::ostringstream oss;
+    oss.imbue(std::locale::classic());
     format(oss, fmt.c_str(), args...);
     return oss.str();
```

Comment:
How does this have any effect? This is pretty much dead code (most of our format strings are c strings, so the other function is called). Might as well remove the code here to clarify this is not the primary implementation.

```suggestion
    return format(fmt.c_str(), args...);
```

Reply:
Ouch, good catch.

Reply:
I've now given all functions the same treatment. Please re-review :)

Reply:
I think this can still be replaced by a single line `return format(fmt.c_str(), args...);`.

Reply:
Fixed!